src/Mapbender/WmsBundle/Resources/translations/messages.en.yaml

```diff
@@ -160,3 +166,36 @@ mb:
         description: DimensionsHandler groups and calls the dimension supported instances at the map
     dimensionsets:
       no_dimension: no dimension
+  core:
+    dimensionset:
+      admin:
+        title: Title
+        group: Group
+        extent: Extent
+    dimensionshandler:
+      admin:
+        tooltip: Tooltip
+        dimensionsets: Dimensionsets
+    wmsloader:
+      admin:
+        defaultformat: Default format
+        defaultinfoformat: Default info format
+    admin:
+      ? ''
+      : vendorspecifictype:
+          name: Name
+          default: Default
+          hidden: Hidden
+      onlineresource:
+        format: Format
+        href: href
+      legendurltype:
+        width: Width
+        height: Height
+        onlineresource: Online resource
+label' => 'mb:
```

Comment:
wrong syntax

label' => 'mb:
  core:
    admin:
      ? ''
      : vendorspecifictype:

Reply:
that is outdated already